src/shared/sentry.js

```diff
@@ -0,0 +1,60 @@
+import * as Sentry from '@sentry/react';
+import {Integrations} from '@sentry/tracing';
+import {SENTRY_DSN} from './consts';
+//import version from '../Version';
+
+export const updateSentryUser = (user) => {
+    Sentry.setUser(user);
+}
+
+export const initSentry = () => {
+    Sentry.init({
+        dsn: `${SENTRY_DSN}`,
+        integrations: [
+            new Integrations.BrowserTracing(),
+        ],
+
+        // We recommend adjusting this value in production, or using tracesSampler
+        // for finer control
+        tracesSampleRate: 1.0,
+    });
+}
+
+export const initSentry2 = () => {
+    Sentry.init({
+        //dsn: `https://${SENTRY_KEY}@sentry.kli.one/2`,
+        dsn: "https://0072ad3d8e9345f685c9b2378119c9cb@sentry.kli.one/3",
```

Comment:
Don't leave sensitive info in git. Maybe remove all function if it's not used

Reply:
done